imap_processing/swe/decom_swe.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+
+from imap_processing import decom
+
+
+def decom_packets(packet_file: str):
+    """Decom SWE data packets using SWE packet definition
+    Parameters
+    ----------
+    packet_file : str
+        Path to data packet path with filename
+
+    Returns
+    -------
+    List
+        List of all the unpacked data
+    """
+    xtce_document = Path("imap_processing/packet_definitions/swe_packet_definition.xml")
```

Comment:
Maybe remove the hard-coded string?

Reply:
I chatted with Laura in person about this.